config/initializers/paperclip_hashed_path.rb

```diff
@@ -0,0 +1,7 @@
+Paperclip.interpolates :hashed_path do |attachment, style|
+  secret = 'Ac}SErVlnyhSty9TbEaWb^uedgoFqdKWZhwyKqVT'
```

Comment:
whoa.  We don't want to be committing this to the public, open source code base.
Shouldn't we be reading this from the ENV if this is `production` ?